bar-trender-front-end/src/views/ErrorPage.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import DeviceIdentifier from 'react-device-identifier';
+
+// reactstrap components
+import {
+  Input,
+  InputGroupAddon,
+  InputGroupText,
+  InputGroup,
+  Container,
+  Row,
+  Col,
+} from "reactstrap";
+
+// core components
+import MainNavbar from "../components/Navbars/MainNavbar.js";
+import LandingPageHeader from "../components/Headers/LandingPageHeader.js";
+import DefaultFooter from "../components/Footers/DefaultFooter.js";
+
+// images
+import image_left from '../assets/img/expositions/hU-kQ3Epxeq2dhaBpUgYfYaPhHEOKXnHXSeUqLjTygYBV05OHhUSZEWilh_Da9zkI1d_cgz91KIPevD_BBhBWhaKevognkx6Bv7-QwkQdRG9oznKG6wOae4avH8ksi6bkJBLWl4.png';
+import image_left_2 from "../assets/img/expositions/Yn0xRl4G5E1eabgf9nyC9j6DVQVHd5DBNcPehVZwakLHYP-toRbW22a8kFesYK_taX0ZY_WviWVcT3bQ40tlKhaKSuAQAu6graIF.png";
+import image_right from "../assets/img/expositions/TR9IDnSgMV79XktfRCxesUmLacTZJI9fb3Cv3-aMamIGyWdL_OagKWYcJJAPqgm62bjW9I6yHlMsOhowVROsAUiNui0CGo-qmPU-.png";
+import employee_0 from "../assets/img/carlos.png";
+import employee_2 from '../assets/img/victor.png';
+import employee_3 from '../assets/img/enrique.png';
+import employee_4 from '../assets/img/alvaro.png';
+import employee_5 from '../assets/img/jose.png';
+import employee_6 from '../assets/img/carlos-pardo.png';
+import employee_7 from '../assets/img/alejandro.png';
+import employee_8 from '../assets/img/xema.png';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7131439)
```suggestion
import employee_8 from "../assets/img/xema.png";
```